x-pack/plugins/ingest/package.json

```diff
@@ -11,6 +11,7 @@
   },
   "dependencies": {
     "@types/color": "^3.0.0",
+    "@types/recompose": "^0.27.0",
```

Comment:
I think they messed up these deps a bit where they have mixed `@types` under deps and `devDeps`. I believe this and the color should move up to `devDeps`. This is still approved for check-in. I made a separate issue for this:

https://github.com/elastic/ingest-dev/issues/62